pkg/operator/ceph/object/objectstore_test.go

```diff
@@ -199,3 +242,41 @@ func TestGetObjectBucketProvisioner(t *testing.T) {
 	bktprovisioner = GetObjectBucketProvisioner(context, testNamespace)
 	assert.Equal(t, bucketProvisionerName, bktprovisioner)
 }
+
+func TestDashboard(t *testing.T) {
+	logger.Info("Testing dash board")
```

Comment:
```suggestion
	logger.Info("Testing dashboard")
```

Reply:
Actually, this is a unit test. Why is this message even in here? Remove.

Reply:
Done